src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/KnownPseudoHeaders.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Net.Http.HPack
+{
+    internal static class KnownPseudoHeaders
+    {
+        internal static class Names
+        {
+            public static readonly byte[] Method = HPackEncoder.EncodeHeaderName(":method");
```

Comment:
What we really should do here is use the HPACK static header table, then we don't need to encode these at all.

I have a change that does this in part; if you want to grab it and merge you are welcome to, or we can figure out the best way to coordinate this.

Reply:
Done.